apps/explorer/src/app/components/txs/details/tx-order.tsx

```diff
@@ -39,7 +39,7 @@ export const TxDetailsOrder = ({
 
   return (
     <>
-      <TableWithTbody className="mb-8">
+      <TableWithTbody className="mb-8" allowWrap={true}>
```

Comment:
suggestion (non blocking): Maybe make wrap the default behaviour if most tables need it

Reply:
I agree - I'd like to make it default, but currently more views require it than don't. I'll invert it after fixing the blocks page (which comes after all the TX views)